tests/aequilibrae/paths/test_pathResults.py

```diff
@@ -174,3 +174,24 @@ def test_update_blocking_trace(self):
         self.r.update_trace(6)
         self.assertEqual(list(self.r.path_nodes), [4, 1, 3, 6])
         self.assertEqual(list(self.r.path), [4, 1, 6])
+
+
+class TestCentroidsLast(TestCase):
+    def test_compute_paths_centroid_last_node_id(self):
+        # Issue 307 consisted of not being able to compute paths between two
+        # centroids if we were skimming and one of them had the highest node_id
+        # in the entire network
+        zipfile.ZipFile(st_varent_network).extractall(gettempdir())
+        self.st_varent = join(gettempdir(), 'St_Varent')
+        self.project = Project()
+        self.project.open(self.st_varent)
+        self.project.network.build_graphs()
+        self.g = self.project.network.graphs["c"]  # type: Graph
+        self.g.set_graph("distance")
+        self.g.set_skimming('distance')
+
+        self.r = PathResults()
+        self.r.prepare(self.g)
+
+        self.r.compute_path(387, 1067)
```

Comment:
did you want to assert what the computed path actually is?

Reply:
Not really. The only concern is it actually running